package.json

```diff
@@ -130,7 +130,7 @@
 		"test-server:coverage": "npm run -s test-server -- --coverage",
 		"test-server:watch": "npm run -s test-server -- --watch",
 		"translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -k translate,__,_x,_n,_nx -e date '**/*.js' '**/*.jsx' '**/*.ts' '**/*.tsx' '!build/**' '!packages/**/dist/**' '!public/**' '!**/test/**' '!node_modules/**' '!apps/full-site-editing/**' '!**/node_modules/**'",
-		"typecheck": "tsc --noEmit",
+		"tsc": "NODE_OPTIONS='--max-old-space-size=4096' tsc",
```

Comment:
this isn't working for me without `--project client` (I saw you just removed it on another suggestion 😂) when I try to run `npm run tsc` directly I get the help report for `tsc`

Reply:
I restored the `typecheck` script.